config/locales/es.yml

```diff
@@ -3173,35 +3150,35 @@ es:
           title: Parece que tu estado civil es "Cónyuge sobreviviente calificado"
       nc_retirement_income_subtraction:
         edit:
-          bailey_description: 'El Acuerdo Bailey es una decisión de la Corte Suprema de Carolina del Norte. Dice que algunos planes de jubilación no pueden pagar impuestos en Carolina del Norte. Estos planes incluyen:'
-          bailey_more_info_html: Visite <a target="_blank" href="https://www.ncdor.gov/bailey-decision-concerning-federal-state-and-local-retirement-benefits">el Departamento de Ingresos de Carolina </a> del Norte para obtener más información.
+          bailey_description: El Acuerdo Bailey es un fallo de la Corte Suprema de Carolina del Norte que establece que ciertos planes de jubilación son libres de impuestos en Carolina del Norte.
+          bailey_more_info_html: Visita el <a target="_blank" href="https://www.ncdor.gov/bailey-decision-concerning-federal-state-and-local-retirement-benefits">Departamento de Ingresos de Carolina del Norte</a> for more information.
           bailey_reveal_bullets:
-          - Sistema de Jubilación de Maestros y Empleados Estatales de Carolina del Norte
-          - Sistema de jubilación de empleados del gobierno local de Carolina del Norte
-          - Sistema Consolidado de Jubilación Judicial de Carolina del Norte
-          - Sistema de Retiro de Empleados Federales
-          - Sistema de jubilación del servicio civil de los Estados Unidos
-          bailey_settlement_at_least_five_years: "¿Tenía usted o su cónyuge al menos cinco años de servicio acreditable antes del 12 de agosto de 1989?"
-          bailey_settlement_checkboxes: Marque todas las casillas sobre el Acuerdo Bailey que se apliquen a usted o a su cónyuge.
-          bailey_settlement_from_retirement_plan: "¿Recibió usted o su cónyuge beneficios de jubilación del plan 401(k) o 457 de Carolina del Norte y estuvo contratado o contribuyó al plan antes del 12 de agosto de 1989?"
+          - Sistema de Jubilación de Maestros y Empleados del Estado de Carolina del Norte
+          - Sistema de Jubilación de Empleados Gubernamentales Locales de Carolina del Norte
+          - Sistema de Jubilación Judicial Consolidado de Carolina del Norte
+          - Sistema de Jubilación de Empleados Federales
+          - Sistema de Jubilación del Servicio Civil de los Estados Unidos
+          bailey_settlement_at_least_five_years: " ¿Prestaste tú o tu cónyuge prestaron al menos cinco años de servicio acreditable antes del 12 de agosto de 1989?"
```

Comment:
ugh need to probably remove the leading space? dont want to block though

Reply:
fixed in transifex